docs/v0.5/runtimes/streaming.md

```diff
@@ -116,3 +103,49 @@ riff streaming processor create time-averager \
   --input in \
   --output out
 ```
+
+## Testing with dev-utils
+
+The [dev-utils](https://github.com/projectriff/dev-utils/) container offers CLI utilities which can be used to publish messages to an input stream and subscribe to messages on an output stream.
+
+To run dev-utils in a pod in the `default` namespace, using a service account called `dev-utils`:
+
+```bash
+kubectl create serviceaccount dev-utils
+kubectl create rolebinding dev-utils-edit --clusterrole=edit --serviceaccount=default:dev-utils
+kubectl run dev-utils --serviceaccount=dev-utils --generator=run-pod/v1 --image=projectriff/dev-utils
+```
+
+Using separate terminal windows, subscribe to the `in` and `out` streams.
+
+```bash
+kubectl exec dev-utils -it -- subscribe in --payload-encoding raw
```

Comment:
if we have people subscribe to `in` we should do something with it

Reply:
We publish to it - I think it's reassuring to see the things that you publish from the CLI appear somewhere.

Reply:
Then a quick note saying that the user should expect to see the messages they publish in this console would help. Right now there's no context for what the command does or what is expected.